### PR TITLE
asgguylaststand1 fix

### PR DESCRIPTION
**Pull request by funnymanDELUXE**
alright well i edited the file and shit but it for some reason is saying its mass deleted and mass created a bunch of things when i didnt, but oh well. all i changed was the death was shifted to a new bit of code where it spawns the deadasgguyleg actor

### CODE REVIEW

**Comment by pa1nki113r** [comment]
What exactly are the changes in this PR? Can you summarize what's in this?

**Comment by scaryred24** [comment]
Just looking back on the discord server it seems that the auto shotty enemy goes invisible during its last stand animations. Don't believe a last stand was ever implemented let alone planned at this current stage of development. 

For next time can you add a comment during a push or at least give us a better explanation of what you tried to do here? It would be appreciated rather than having the project leader scratching his head wondering what this is all about.

**Comment by funnymanDELUXE** [comment]
yeah sorry about that, i was having issues with this and honestly i was kinda getting frustrated lol. all it does is add a death state for the asgguy after he goes into asgguylaststand1, adding "
		Death:
         	TNT1 A 1 A_SpawnItem ("DeadASGGuy_NoLeg", 5)
       		Stop"
before the last stand bit. again, sorry for the confusion, ill be sure to explain it in more detail next time 👌

**Comment by pa1nki113r** [comment]
All good, I'll approve it when I get to merge some of my changes either tonight or tomorrow
